dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteItemPresenter.java

```diff
@@ -80,7 +87,7 @@ public void initialize(final URL location, final ResourceBundle resources) {
             });
 
         } catch (final Exception e) {
-            LOG.error("Encountered an error making the delete file presenter", e);
+            LOG.error("Encountered an error making the delete file presenter. \n{}", e);
```

Comment:
This is not the correct formatting for an error message.  If you want the message, then you need to use string concatenation.